docs/sources/alerting/unified-alerting/opt-in.md

```diff
@@ -1,12 +1,12 @@
 +++
-title = "Opt-in to Grafana 8 alerting"
-description = "Enable Grafana 8 Alerts"
+title = "Opt-in to Grafana alerting"
+description = "Enable Grafana alerts"
 weight = 115
 +++
 
-# Opt-in to Grafana 8 alerting
+# Opt-in to Grafana alerting
 
-This topic describes how to opt-in to Grafana 8 alerting and the rules and restrictions that govern the migration of existing dashboard alerts to the new alerting system. You can [disable Grafana 8 alerts]({{< relref "./opt-in.md#disable-grafana-8-alerts" >}}) and use the legacy dashboard alerting if needed.
+Grafana alerting is enabled by default for new OSS installations. For older installations, it is still an [opt-in]({{< relref "./unified-alerting/opt-in.md" >}}) feature. This topic describes how to opt-in to Grafana alerting if you have an existing Grafana installation and the rules and restrictions that govern the migration of existing dashboard alerts to the new alerting system. You can [disable Grafana 8 alerts]({{< relref "./opt-in.md#disable-grafana-8-alerts" >}}) and use the legacy dashboard alerting if needed.
```

Comment:
I wonder if it is better to say something like "Grafana alerting is enabled by default unless you already use legacy alerting"

Reply:
You cannot have legacy alerting for new installs as of 8.3, correct? Keeping the two sentences as is, to avoid confusion.